cryodrgn/source.py

```diff
@@ -0,0 +1,278 @@
+import os.path
```

Comment:
Am I correct that the goal is for this to eventually replace all the `parse_xxx` and `LazyImage` code? That's cool.

Reply:
Yes that was my goal here.